libs/frontend/feature-build/src/components/build-render/ProseMirrorEditor.vue

```diff
@@ -100,7 +100,10 @@ watch(gradientColorStyle, () => {
 })
 
 onBeforeUnmount(() => {
-  if (editor.value) {
+  if (editor.value?.editView) {
+    if (editor.value.editView.state.doc.content.size <= 2) {
```

Comment:
I noticed that if you add some text in a container/footer/etc, then delete it with the prosemirror editor, we can't drag and drop into that component anymore. The reason is that there is an empty `<div></div>` wrapper, which counts as content, so the drop logic fails. This code completely clears the content if the editor is empty.

Another solution could be to allow dropping into components with content, or detecting that the content is "empty."